db/view/view.cc

```diff
@@ -1058,8 +1058,8 @@ query::clustering_row_ranges calculate_affected_clustering_ranges(const schema&
         const mutation_partition& mp,
         const std::vector<view_and_base>& views,
         gc_clock::time_point now) {
-    std::vector<nonwrapping_range<clustering_key_prefix_view>> row_ranges;
-    std::vector<nonwrapping_range<clustering_key_prefix_view>> view_row_ranges;
+    utils::chunked_vector<nonwrapping_range<clustering_key_prefix_view>> row_ranges;
```

Comment:
Just a thought (feel free to ignore) - do we really need an indexable vector in all the places we use chunked_vector, including here? We also have Seastar's chunked_fifo if we don't. Unfortunately chunked_vector doesn't begin with a comment analyzing how it's different than chunked_fifo.